gwpy/utils/decorators.py

```diff
@@ -112,10 +111,6 @@ def wrapped(*args, **kwargs):
                         func.__name__, returntype.__name__, str(exc)),
                 )
                 raise
-        try:
-            return wraps(func)(wrapped)
-        except AttributeError:  # python < 3.0.0
-            wrapped.__doc__ == func.__doc__
-            return wrapped
+        return wraps(func)(wrapped)
```

Comment:
@alurban, this should just be I think)...:

```python
        @wraps(func)
        def wrapped(*args, **kargs):
            ...
```

rather than calling `wraps` as a function. You can also remove the entire comment about wraps.

Reply:
@duncanmmacleod, I was hoping you'd catch that one, I tried to figure it out last night but had gotten stuck. Your suggestion appears to work for me locally, we'll find out soon whether build tests pass